### PR TITLE
Add NRQL Alert Condition Resource

### PR DESCRIPTION
**Pull request by dthorsen**
I wasn't sure what to add to the CHANGELOG.md. Is there a desired format for the Changelog entries? Since this was a feature addition I bumped the minor version but wasn't sure how releases will be handled for these providers since they can now release out of cycle with terraform core.

Thanks in advance!

```
$ make testacc TEST=./newrelic TESTARGS='-run=TestAccNewRelicNrqlAlertCondition_Basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./newrelic -v -run=TestAccNewRelicNrqlAlertCondition_Basic -timeout 120m
=== RUN   TestAccNewRelicNrqlAlertCondition_Basic
--- PASS: TestAccNewRelicNrqlAlertCondition_Basic (4.71s)
PASS
ok  	github.com/terraform-providers/terraform-provider-newrelic/newrelic	5.244s
```

### CODE REVIEW

**Comment by dthorsen** [comment]
@RubyLi0612 I made those adjustments. Thanks again for reviewing the code!

**Comment by err0** [comment]
what is holding up the merge on this?

File: newrelic/resource_newrelic_nrql_alert_condition.go [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+	log.Printf("[INFO] Creating New Relic NRQL alert condition %s", condition.Name)
+
+	condition, err := client.CreateAlertNrqlCondition(*condition)
+	if err != nil {
+		return err
+	}
+
+	d.SetId(serializeIDs([]int{condition.PolicyID, condition.ID}))
+
+	return nil
```
Comment (grubernaut) [inline newrelic/resource_newrelic_nrql_alert_condition.go]: The `Create` function here should return the `Read` function, to catch any `Computed` values, or any configuration drift after creation.

Comment (dthorsen) [inline newrelic/resource_newrelic_nrql_alert_condition.go]: This has been modified to read the API per your recommendation. Acceptance tests re-ran successfully:
make testacc TEST=./newrelic TESTARGS='-run=TestAccNewRelicNrqlAlertCondition_Basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./newrelic -v -run=TestAccNewRelicNrqlAlertCondition_Basic -timeout 120m
=== RUN   TestAccNewRelicNrqlAlertCondition_Basic
--- PASS: TestAccNewRelicNrqlAlertCondition_Basic (7.47s)
PASS
ok  	github.com/terraform-providers/terraform-provider-newrelic/newrelic	8.080s

**Pull request merged**